app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -51,7 +51,7 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
     private val sessionObserver by lazy {
         object : SessionManager.Observer {
             override fun onSessionSelected(session: Session) {
-                serviceLocator.screenController.showBrowserScreenForCurrentSession(supportFragmentManager, session)
+                serviceLocator.screenController.showBrowserScreenForCurrentSession(supportFragmentManager, session, false)
```

Comment:
nit: With unintuitive boolean arguments, I like to include arg labels to improve readability: `showBrowserScreenForCurrentSession(..., menu = false)`